cli/cmd/lib_cluster_config.go

```diff
@@ -290,10 +290,10 @@ func getClusterUpdateConfig(cachedClusterConfig clusterconfig.Config, awsCreds A
 func confirmInstallClusterConfig(clusterConfig *clusterconfig.Config, awsCreds AWSCredentials, awsClient *aws.Client) {
 	eksPrice := aws.EKSPrices[*clusterConfig.Region]
 	operatorInstancePrice := aws.InstanceMetadatas[*clusterConfig.Region]["t3.medium"].Price
-	operatorEBSPrice := aws.EBSMetadatas[*clusterConfig.Region].Price * 20 / 30 / 24
+	operatorEBSPrice := aws.EBSMetadatas[*clusterConfig.Region]["gp2"].PriceGB * 20 / 30 / 24
 	elbPrice := aws.ELBMetadatas[*clusterConfig.Region].Price
 	apiInstancePrice := aws.InstanceMetadatas[*clusterConfig.Region][*clusterConfig.InstanceType].Price
-	apiEBSPrice := aws.EBSMetadatas[*clusterConfig.Region].Price * float64(clusterConfig.InstanceVolumeSize) / 30 / 24
+	apiEBSPrice := aws.EBSMetadatas[*clusterConfig.Region][*clusterConfig.InstanceVolumeType].PriceGB*float64(clusterConfig.InstanceVolumeSize)/30/24 + aws.EBSMetadatas[*clusterConfig.Region][*clusterConfig.InstanceVolumeType].PriceIOPS*float64(*clusterConfig.InstanceVolumeIops)/30/24
```

Comment:
After you remove the default for `clusterConfig.InstanceVolumeIops`, the value will be nil for non-`io1` volumes, so it will probably make sense to check that before adding the IOPS price, e.g. (pseudocode): `if storage type is io1` and/or `if iops != nil`